### PR TITLE
[Proposal] Changes to the documentation of "TYPO3 customize"

### PR DESCRIPTION
**Pull request by rowild**
UPDATE 2020-09-09:
Finished proofreading of complete "customize" document.
Added images and new list element for "deactivating, deleting... shops" (part of that content should actually better go to the manual, esp. the deleting part IMO. But since cache is involved, maybe we can mention these topics here, too?)

------

2020-09-08:
This is incomplete and intended for discussion and feedback only!

This commit includes changes up to and including "Multiple shops". The changes are quite far-reaching and might not reflect what you would like to present here, Norbert, which is, why I prefer to stop here and ask for your feedback, before I continue.

Especially the changes to "Multiple Shop" are extensive. IMO the descriptions there are way to complicated, in the case of pageTS to TYPO3-specific (this description does a better job then the TYPO3 docu! :-) ), and do not really describe what has to be done to achieve a multiple shop setup. Therefore I propose a "follow these steps" solution.

### CODE REVIEW

File: src/typo3/customize.md [hunk 1]
```diff
@@ -198,18 +200,18 @@ plugin.tx_aimeos.settings.i18n.cz.0 {
...
 ```
 
-Using them in TypoScript is more difficult, because TypoScript doesn't allow new lines in values. Therefore, you have to use "\n" (a backslash and the character "n") as replacement:
+Using them in TypoScript is more difficult because TypoScript doesn't allow new lines in values. Therefore, you have to use "\n" (a backslash and the character "n") as replacement:
 
 ```typoscript
 plugin.tx_aimeos.settings.i18n.de.0 {
   domain = client
-  string = The payment was canceled.\nDo you wish to retry your order?
+  string = The payment was cancelled.\nWould you like to place your order again?
```
Comment (aimeos) [inline src/typo3/customize.md:214]: This is a string from the code which shouldn't be changed here

File: src/typo3/customize.md [hunk 2]
```diff
@@ -306,86 +308,105 @@ by this Fluid template:
...
+In *Aimeos* terms, a "shop" is a "site". Therefore, when discussing whether the system can handle "mutlipe shops", we say that *Aimeos* is *multi-site capable*. It allows storing several shops in one database.
 
-## Create new site
+In TYPO3, each shop needs its own TYPO3 page tree, the root of which has to be configured with a special pageTS setting (except the first one).
 
-In the Aimeos configuration settings within the extension manager, you can enter a site code for which a new site will be created. If it already exists, it will get updated with the required entries for the used extension version.
+Follow these steps to create multiple shops/sites:
 
-![Aimeos extension settings](Aimeos-settings.png)
+1. **Create a new TYPO3 page**
```
Comment (aimeos) [inline src/typo3/customize.md:317]: This will create an order list which isn't what you want in that case I think

File: src/typo3/customize.md [hunk 3]
```diff
@@ -306,86 +308,105 @@ by this Fluid template:
...
-* Admin Tools::Settings:
-    1. Search for "aimeos"
-    1. Enter a new site code in the corresponding input field
-    1. Save and return
-* Admin Tools::Extensions:
-    1. Click on the update icon of the Aimeos extension
+    Each shop needs its individual TYPO3 page tree section. Therefore, create a new page that is located outside of any other potential *Aimeos* site that might already exist.
 
-You will see a list of checks that have been done and at the end the types of data that has been added to the new site.
+2. **Create the new site in the Extension Manager**
```
Comment (aimeos) [inline src/typo3/customize.md:321]: Same ordered list here

File: src/typo3/customize.md [hunk 4]
```diff
@@ -306,86 +308,105 @@ by this Fluid template:
...
 
-![PageTS for multiple sites](Aimeos-pagets.png)
+    !!! warning
+        If you update Aimeos from a previous version, you need to run the update script for all sites you've created! Otherwise, required records may be missing and existing data isn't migrated.
 
-* Web::Page
-    1. Select second page tree
-    1. Edit page
-    1. Tab *Resources*, section *TypoScript Configuration*
+3. **Create the new "Aimeos locale"**
```
Comment (aimeos) [inline src/typo3/customize.md:339]: Same here

File: src/typo3/customize.md [hunk 5]
```diff
@@ -306,86 +308,105 @@ by this Fluid template:
...
+    tx_aimeos.mshop.locale.site = <code of site>
+    ```
 
-```typoscript
-plugin.tx_aimeos.settings.mshop.locale.site = <code of site>
-```
+    Click on the icon for *Save and close document* at the top to store the change. Repeat these steps for each page tree where an different shop site other than the "default" site should be used. Clear all TYPO3 caches when you are done.
 
-Click on the icon for *Save and close document* at the top to store the change. Repeat these steps for each page tree where an other shop site other than the "default" site should be used. Clear all TYPO3 caches when you are done.
+5. **TypoScript config**
```
Comment (aimeos) [inline src/typo3/customize.md:360]: Would be better to use headlines (## TypoScript config) instead ordered lists with only one element

File: src/typo3/customize.md (hunk 2 above)
Comment (rowild) [reply src/typo3/customize.md:317]: Quite the opposite: a ordered list as a step-by-step guide after the text "Follow these steps to ...".

File: src/typo3/customize.md (hunk 1 above)
Comment (rowild) [reply src/typo3/customize.md:214]: OK. (But these strings will be fixed, too, and then we have to come back again...)

File: src/typo3/customize.md (hunk 5 above)
Comment (rowild) [reply src/typo3/customize.md:360]: This is the 5th in a list of 6 (or 7?) elements. (The texts below a list element are indented.)

**Pull request merged**